tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,180 @@
+        
+        settings.open_themes()
+        self.take_screenshot('themes')
+        settings.return_to_prev_menu(settings.screen, gaia_header=False)
+        
+        settings.open_addons()
+        self.take_screenshot('addons')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_achievements()
+        self.take_screenshot('achievements')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_firefox_accounts()
+        self.take_screenshot('firefox_accounts')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_findmydevice()
+        self.take_screenshot('findmydevice')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_screen_lock()
+        self.take_screenshot('screen_lock')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_app_permissions()
+        self.take_screenshot('app_permissions')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_do_not_track()
+        self.take_screenshot('do_not_track')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_browsing_privacy()
+        self.take_screenshot('browsing_privacy')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_media_storage()
+        self.take_screenshot('media_storage')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_application_storage()
+        self.take_screenshot('application_storage')
+        settings.return_to_prev_menu(settings.screen)
+
+        ### Device Information and its sub pages
+        device_info_page = settings.open_device_info()
+        self.take_screenshot('dev_info')
+        GaiaImageCompareTestCase.scroll(self.marionette, 'down', device_info_page.screen.size['height'],
+                                        screen=device_info_page.screen)
+        self.take_screenshot('dev_info')
+
+        # Enter more information dialog
+        moreinfo_page = device_info_page.tap_more_info()
+        self.take_screenshot('dev_info-more_info')
+        GaiaImageCompareTestCase.scroll(self.marionette, 'down', device_info_page.screen.size['height'],
+                                        screen=moreinfo_page.screen)
+        self.take_screenshot('dev_info-more_info')
+        settings.return_to_prev_menu(device_info_page.screen)
+
+        # Enter reset phone dialog
+        device_info_page.tap_reset_phone()
+        self.take_screenshot('dev_info-reset')
+        device_info_page.confirm_reset(False)
+
+        # Entering system view, need to switch frame
+        device_info_page.tap_update_frequency()
+        self.take_screenshot('dev_info-update-freq')
+        device_info_page.exit_update_frequency()
+        settings.return_to_prev_menu(settings.screen)
+
+        ### Downloads page
```

Comment:
same reason as adding '### Device Information and its sub pages' comment - should we put a clear divider between accessing each menu for readability?